parachain-template/node/src/command.rs

```diff
@@ -45,7 +45,7 @@ impl SubstrateCli for Cli {
 		"Parachain Collator Template\n\nThe command-line arguments provided first will be \
 		passed to the parachain node, while the arguments provided after -- will be passed \
 		to the relay chain node.\n\n\
-		parachain-collator <parachain-args> -- <relay-chain-args>"
+		parachain-template-node <parachain-args> -- <relay-chain-args>"
```

Comment:
Nit (feel free to ignore). Maybe you can use the same technique that has been used here

https://github.com/paritytech/cumulus/blob/17ca448df53c76556ff85538f59f850c3fdd6f01/polkadot-parachain/src/command.rs#L179-L186

Reply:
Will do